nightly-main/rhel-ubi/9/Dockerfile

```diff
@@ -0,0 +1,66 @@
+FROM redhat/ubi9
+
+LABEL maintainer="Swift Infrastructure <swift-infrastructure@forums.swift.org>"
+LABEL description="Docker Container for the Swift programming language"
+
+RUN yum install -y  \
+  git               \
+  gcc-c++           \
+  libcurl-devel     \
+  libedit-devel     \
+  libuuid-devel     \
+  libxml2-devel     \
+  ncurses-devel     \
+  python3-devel     \
+  rsync             \
+  sqlite-devel      \
+  unzip             \
+  zip
+
+# Everything up to here should cache nicely between Swift versions, assuming dev dependencies change little
+
+# gpg --keyid-format LONG -k FAF6989E1BC16FEA
+# pub   rsa4096/FAF6989E1BC16FEA 2019-11-07 [SC] [expires: 2021-11-06]
```

Comment:
This looks already expired? `[expires: 2021-11-06]`

Reply:
Fixed the copy/paste issue